### PR TITLE
chore: add community_meeting_4.20 

### PR DESCRIPTION
**Pull request by cqqqq777**
#### Check the PR title.

- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.
- [ ] Attach the PR updating the user documentation if the current PR requires user awareness at the usage level. [User docs repo](https://github.com/cloudwego/cloudwego.github.io)

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

File: meeting_notes/CloudWeGo_community_meeting_4.20.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+1. kitex 新版本，地址：https://github.com/cloudwego/kitex/releases/tag/v0.5.2
+2. 在以往的 kitex 版本中针对超时都会有一个默认的重试，新版本中为重试添加了一个新的配置项，允许用户对超时默认不进行重试。
+3. 在代码生成工具方面：支持了 windows 系统用户使用 kitex 生成代码。
+4. RPC time out：在以往的版本中对超时的错误码只有一个 RPC time out 的超时，现在支持更细粒度的超时错误码，如可以分辨出在 context 中设置的 deadline 或 cancel 方法。
+5. thrift fast codec：新增对 unknown fields 的序列化与反序列化。
+
+---
+
+### 议程二：shmipc-go 项目解读 & RoadMap 分享 @李万治
```
Comment (GuangmingLuo) [inline meeting_notes/CloudWeGo_community_meeting_4.20.md:17]: 万治 -> goodbye-babyer

Comment (cqqqq777) [reply meeting_notes/CloudWeGo_community_meeting_4.20.md]: ok

File: meeting_notes/CloudWeGo_community_meeting_4.20.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+**参会人** ：GuangmingLuo, goodbye-babyer, zhquzzuli, welkeyever, YangruiEmma, halst, whalecold, Cr., Haswf, li-jin-go, Joway, Duslia, FGYFFFF, ppzqh, Marina-Sakai, Claude-zq, skyenought, rogerogers, L2ncE, Nihilism , Felix, shiningstoned, Yuqin425, cqqqq777
+
+**会前必读** ：http://www.cloudwego.io/ ; https://github.com/cloudwego
+
+### 议程一：kitex 新版本变更解读 @ppzqh
+
+1. kitex 新版本，地址：https://github.com/cloudwego/kitex/releases/tag/v0.5.2
+2. 在以往的 kitex 版本中针对超时都会有一个默认的重试，新版本中为重试添加了一个新的配置项，允许用户对超时默认不进行重试。
+3. 在代码生成工具方面：支持了 windows 系统用户使用 kitex 生成代码。
+4. RPC time out：在以往的版本中对超时的错误码只有一个 RPC time out 的超时，现在支持更细粒度的超时错误码，如可以分辨出在 context 中设置的 deadline 或 cancel 方法。
```
Comment (ppzqh) [inline meeting_notes/CloudWeGo_community_meeting_4.20.md:12]: 「超时错误码」 -> 「超时错误类型」

**Comment by Haswf** [comment]
LGTM. Many thanks

**Comment by Claude-Zq** [comment]
LGTM

**Review by goodbye-babyer — DISMISSED** [review]

**Review by GuangmingLuo — APPROVED** [review]

**Pull request merged**